easybuild/easyconfigs/f/filevercmp/filevercmp-20191210-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,33 @@
+# Updated: Denis Kristak (INUITS)
+easyblock = 'ConfigureMake'
+
+name = 'filevercmp'
+version = '20191210'
+local_commit = 'df20dcc'
+
+homepage = 'https://github.com/ekg/filevercmp'
+description = """filevercmp function as in sort --version-sort."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+github_account = 'ekg'
+source_urls = [GITHUB_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+patches = ['%(name)s-20191210_build-libs.patch']
+checksums = [
+    '89835829a7829f7a25783b2cf9d482f1e3c794703343c9214c15c66a8c7f4aae',  # df20dcc.tar.gz
+    '051438f76dd04219abfb283f61101c04d748407031e180b7ae3841344416ec4f',  # filevercmp-20191210_build-libs.patch
+]
+
+builddependencies = [('binutils', '2.36.1')]
+
+skipsteps = ['configure']
+
+installopts = 'DESTDIR="" PREFIX=%(installdir)s '
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s', 'lib/libfilevercmp.%s' % SHLIB_EXT],
+    'dirs': [],
+}
+
```

Comment:
Could we add a sanity command here, which just passes two dummy strings to the tool?

Reply:
@bedroge Done.